assets/js/components/activation/activation-app.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Activation App component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+import { sendAnalyticsTrackingEvent } from 'GoogleUtil';
+
+/**
+ * Internal dependencies
+ */
+import { ActivationMain } from './activation-main';
+import Notification from '../notifications/notification';
+import NotificationCounter from '../notifications/notification-counter';
+
+export class ActivationApp extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			hasError: false,
+		};
+	}
+
+	componentDidCatch( error, info ) {
+		this.setState( {
+			hasError: true,
+			error,
+			info,
+		} );
+	}
+
+	render() {
+		const {
+			hasError,
+			error,
+			info,
+		} = this.state;
+
+		if ( hasError ) {
+			return <Notification
+				id="googlesitekit-error"
+				key="googlesitekit-error"
+				title={ error }
```

Comment:
Should this be using `error.message` ala c063ba95a35ca959efee5db99f660cc5770f0053?

Reply:
Yep, good catch 👍